thirdparty/runtime_environments.go

```diff
@@ -0,0 +1,51 @@
+package thirdparty
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+type RuntimeEnvironmentsClient struct {
+	Client  *http.Client
+	BaseURL string
+	APIKey  string
+}
+
+// getImageNames returns a list of strings containing the names of all images from the runtime environments api
+// TODO: Remove nolint:unused when DEVPROD-6983 is resolved.
+//
+//nolint:unused
+func getImageNames(ctx context.Context, c *RuntimeEnvironmentsClient) ([]string, error) {
```

Comment:
I think it's more common to have it like this:
```suggestion
func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string, error) {
```

You might also want to create a function like `NewRuntimeEnvironmentsClient()` that creates a new client.